packages/back-end/src/services/python.ts

```diff
@@ -17,7 +17,9 @@ type PythonServerResponse<T> = {
 // The stats engine usually finishes within 1 second
 // We use an overly conservative timeout to account for high load
 const STATS_ENGINE_TIMEOUT_MS = 60_000;
-const MAX_POOL_SIZE = 4;
+const MAX_POOL_SIZE = process.env.GB_STATS_ENGINE_POOL_SIZE
+  ? Number(process.env.GB_STATS_ENGINE_POOL_SIZE)
+  : 4;
```

Comment:
nit: I think the other env variables we have are prefixed with the full `GROWTHBOOK`

Reply:
😂  truer words have never been typed -- hoping we can consolidate on a pattern for our env variables in the near future

Reply:
Should we change this to just use the number of CPU cores instead? Do you think there's a benefit to controlling it independently from that?

Reply:
One benefit is that it makes it slightly easier in terraform to calculate the autoscale threshold. Another might be that if we ever have it back on the api-server we might want it to be # cores - 1 or -2 to account for the app and api processes.  self-hosted people could also theoretically want to specify how many they want to use.  They might have 1 core, but still want multiple to run at the same time?  I guess that would just slow them both down to take twice as long because there is probably not much sleeping time in that script.    Anyhow I don't think it hurts to have separate control. In terraform we can always calculate the one from the other so we change it in only one place.